bootstrap.py

```diff
@@ -296,6 +294,12 @@ def apply_configuration(node_set, configuration_file, global_patches):
             "--wait=false",  # We can't wait here, the cluster won't become ready without CNI
         )
 
+        # Wait a few seconds for the nodes to switch into rebooting state
+        time.sleep(5)
+
+        for node in set(to_reboot):
```

Comment:
```suggestion
        for node in to_reboot:
```

No need to make a set here since we don't do set operations.